api/pets/petsRouter.js

```diff
@@ -22,7 +22,8 @@ router.get('/', authRequired, async (req, res) => {
 /******************************************************************************
  *                      GET an existing customers pet by pet id
  ******************************************************************************/
-router.get('/:id', authRequired, async (req, res) => {
+// router.get('/:id', authRequired, async (req, res) => {
```

Comment:
same with this code. If it needs to be commented out for a specific reason, make sure to comment next to it what it will be used for. Otherwise just delete any commented out code that isn't needed.

Reply:
I uncommented out the code as it is important.